roles/couchdb/Dockerfile

```diff
@@ -1,4 +1,2 @@
-FROM klaemo/couchdb:latest
-
-# Configuration
+FROM klaemo/couchdb:1.6.1
 COPY files /usr/local/etc/couchdb
```

Comment:
Do you expose this directory as a Volume? That way, we could mount and perform backups easily.